concourse/scripts/spec/script_testing_helpers.rb

```diff
@@ -0,0 +1,100 @@
+require "aruba/rspec"
+require "pathname"
+
+def find_git_repository_path()
+  path = __FILE__
+  begin
+    if File.exists? File.join(path, '.git')
+      return path
+    end
+    path = File.realpath(File.join(path, '..'))
+  end while path != "/"
+  raise "Cannot find a git repository parent of '#{File.dirname(__FILE__)}'"
+end
+
+def repostory_name()
+  File.basename(find_git_repository_path())
+end
+
+def prepare_aruba_config()
+  RSpec.configure do |config|
+    config.before(:all) do
+      @aruba_orig_workdir = Aruba.config.working_directory
+
+      # We run scripts with git commands, and as we don't want to mess around
+      # with our own repository, we setup the workdir in a different location.
+      #
+      # But Aruba demands it to be relative
+      @workdir = Dir.mktmpdir("paas-cf-tmpdir-")
+      relative_workdir = Pathname.new(@workdir).relative_path_from(Pathname.pwd)
+
+      Aruba.configure do |config|
+        config.working_directory = relative_workdir
+        config.home_directory = @workdir
+        @repo_root = find_git_repository_path()
+
+        config.command_search_paths = [
+          File.join(@repo_root, "concourse", "scripts"),
+          File.join(@repo_root, "manifests", "scripts")
```

Comment:
I guess we could exclude manifests for the moment because they already have a test framework. Maybe make the path relative to `__FILE__` for the moment?